docs/extras/use_cases/tabular/index.mdx

```diff
@@ -4,32 +4,384 @@ sidebar_position: 1
 
 # Analyzing structured data
 
+## Use case
 Lots of data and information is stored in tabular data, whether it be csvs, excel sheets, or SQL tables.
-This page covers all resources available in LangChain for working with data in this format.
+In this page, we will cover SQL use case.
 
-## Document loading
-If you have text data stored in a tabular format, you may want to load the data into a Document and then index it as you would
-other text/unstructured data. For this, you should use a document loader like the [CSVLoader](/docs/modules/data_connection/document_loaders/how_to/csv.html)
-and then you should [create an index](/docs/modules/data_connection) over that data, and [query it that way](/docs/use_cases/question_answering/how_to/vector_db_qa.html).
+SQL Chains and Agents can be used to **build and run** SQL queries based on natural language prompts. LangChain is compatible with any SQL dialect supported by **SQLAlchemy**, such as MS SQL, MySQL, MariaDB, PostgreSQL, Oracle SQL, Databricks, and SQLite.
 
-## Querying
-If you have more numeric tabular data, or have a large amount of data and don't want to index it, you should get started
-by looking at various chains and agents we have for dealing with this data.
+Some use cases may be:
 
-### Chains
+- Generating queries that will be run based on natural language questions
+- Creating chatbots that can answer questions based on database data
+- Building custom dashboards based on insights a user wants to analyze
 
-If you are just getting started, and you have relatively small/simple tabular data, you should get started with chains.
-Chains are a sequence of predetermined steps, so they are good to get started with as they give you more control and let you 
-understand what is happening better.
+## Overview
 
-- [SQL Database Chain](/docs/use_cases/tabular/sqlite.html)
+LangChain provides a set of tools for any user to interact with SQL Databases in many ways. We can:
 
-### Agents
+1. `Build SQL queries` based on user questions
+2. `Obtain insights from an SQL database` using the SQL Database Chain for handling both query creation and execution
+3. `Interact with a complex SQL database in a robust way` using the SQL Agent which contains a set of tools which make it more robust and flexible when creating and running queries
 
-Agents are more complex, and involve multiple queries to the LLM to understand what to do.
-The downside of agents are that you have less control. The upside is that they are more powerful,
-which allows you to use them on larger databases and more complex schemas. 
+## Quickstart
```

Comment:
This should include DB creation and a chain like `SQLDatabaseChain` that goes end-to-end from query to insight.

Reply:
Great idea, I have just added it to the **quickstart** section